packages/ui/src/components/va-avatar/VaAvatar.vue

```diff
@@ -86,6 +125,11 @@ export default defineComponent({
   vertical-align: var(--va-avatar-vertical-align);
   border-radius: var(--va-avatar-border-radius);
   font-family: var(--va-font-family);
+  background-color: v-bind('backgroundColorComputed');
```

Comment:
We heed quotes here only in case of nested value (ex: backgroundColorComputed.secondary).

Reply:
Yeah, I understand it. But in my opinion will be nice to decide how we will format it all over the project for better consistency.